go.mod

```diff
@@ -5,10 +5,10 @@ go 1.13
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.4.17
-	github.com/Microsoft/hcsshim v0.8.18
+	github.com/Microsoft/hcsshim v0.8.23
 	github.com/agext/levenshtein v1.2.3
 	github.com/containerd/console v1.0.2
-	github.com/containerd/containerd v1.5.3
+	github.com/containerd/containerd v1.5.8-0.20211112185055-5fde9a5b31df
```

Comment:
1.5.8 was tagged now; https://github.com/containerd/containerd/releases/tag/v1.5.8

Reply:
I'll do it separately